config/clusters/meom-ige/support.values.yaml

```diff
@@ -0,0 +1,28 @@
+prometheusIngressAuthSecret:
+  enabled: true
+
+prometheus:
+  server:
+    ingress:
+      enabled: true
+      hosts:
+        - prometheus.meom-ige.2i2c.cloud
+      tls:
+        - secretName: prometheus-tls
+          hosts:
+            - prometheus.meom-ige.2i2c.cloud
+    resources:
+      requests:
+        cpu: 500m
+        memory: 512Mi
+      limits:
+        cpu: 500m
+        memory: 512Mi
```

Comment:
This probably needs to be higher. Let's start with 1G, and then look at how much it's actually using?